client/src/ui/organisms/top-navigation-main/index.tsx

```diff
@@ -32,6 +32,7 @@ export const TopNavigationMain = ({ isOpenOnMobile }) => {
       <MainMenu isOpenOnMobile={isOpenOnMobile} />
 
       <Search
+        id="top-nav-search"
```

Comment:
The skip to search link in the ally nav uses `#main-q`

Reply:
So, we either need to update the a11y nav link or use the same id here.

Reply:
👍  good catch

Reply:
🙈 that link is broken already…